src/actix/actix_telemetry.rs

```diff
@@ -40,6 +40,24 @@ where
             .match_pattern()
             .unwrap_or_else(|| "unknown".to_owned());
         let request_key = format!("{} {}", request.method(), match_pattern);
+        let collection_name = request
+            .match_info()
+            .get("name")
+            .map(ToString::to_string)
+            .or_else(|| {
+                if match_pattern.starts_with("/collections/{name}") {
```

Comment:
I don't think naming pattern is universal for all APIs. If we introduce a new API, it is not enforced that it will use the same parameter name. It is very likely that we will forget to make it matching in codebase this large.